tests/test_zmanim.py

```diff
@@ -73,23 +78,25 @@ def test_using_tzinfo(self):
 
         assert Zmanim(date=day, location=location_tz_str).zmanim[
             "first_stars"
-        ].time() == datetime.time(19, 48)
+        ].time() == datetime.time(19, 45 if _ASTRAL else 48)
 
         assert Zmanim(date=day, location=location).zmanim[
             "first_stars"
-        ].time() == datetime.time(19, 48)
+        ].time() == datetime.time(19, 45 if _ASTRAL else 48)
 
     # Times are assumed for NYC.
     CANDLES_TEST = [
-        (dt(2018, 9, 7, 13, 1), 18, dt(2018, 9, 7, 19, 4), False),
-        (dt(2018, 9, 7, 19, 4), 18, dt(2018, 9, 7, 19, 4), True),
+        (dt(2018, 9, 7, 13, 1), 18, dt(2018, 9, 7, 19, 0 if _ASTRAL else 4), False),
```

Comment:
I think a better option might be in the test to do something similar to the following:

```python
offset = timedelta(minutes=5) if _ASTRAL else 0
assert candle_lighting_time - offset <= actual <= candle_lighting_time + offset
```
So next time we can actually take times from calendars and it should not fail.

Same goes for the other tests.

Reply:
Done